examples/deck.gl/pointcloud/app.js

```diff
@@ -81,30 +82,54 @@ export default class App extends PureComponent {
       selectedCategory,
       selectedExample,
       pointsCount: null,
-      points: null,
+      pointsAttribute: null,
+      colorsAttribute: null,
       loadTimeMs: undefined
     });
     load(uri).then(this._onLoad.bind(this));
   }
 
+  _getBounds(attributes) {
```

Comment:
This is great! 

As a small nit, methods that don't access `this` (pure helper functions calculating output based on their input with no side effects) are usually best structured as global helper functions.

--- 

Not for this PR: For the long term: 
- I feel applications should not need to do this
- (too much coding required just to show a simple file - we really want this example to be short and simple to show off how easy it is to use our point cloud loaders). 
- Bounds calculation should happen automatically, either inside the point cloud loaders, or deck.gl (or both).